src/_common/content/content-viewer/components/AppContentViewerBaseComponent.vue

```diff
@@ -53,23 +54,54 @@ function getComponentType(data: ContentObject): any {
 	}
 }
 
-export function renderChildren(childObjects: ReadonlyArray<ContentObject>) {
+export function renderChildren(
```

Comment:
generally I don't break out the general functions and just put everything in the script/setup block to keep things simple. it'll still be super fast since I'm pretty sure the JS engine just optimizes it anyway

Reply:
This function is exported, which is why it's outside of the script/setup block. The helper function above it is also used within this exported function.